src/pages/using-spark/foundations/design-tokens.mdx

```diff
@@ -3,21 +3,62 @@
 ---
 import SassVarTable from '../../../components/SassVarTable';
 
-# Design Tokens
+<h1 class="docs-b-h1 sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
+  Design Tokens
+</h1>
 
-Spark provides design tokens that store our design decisions.
-These tokens can be retrieved in different formats for different devices.
-These variables provide global settings for components in the system.
-For web, almost every aspect of each Spark component is
-configurable with a Sass variable.
+Our Design Tokens are a catalog from which we base all our visual and branding choices.
 
-** Override the Sass settings by: **
-1. Creating a new Sass file.
-2. Importing it before Spark's main Sass file.
-3. Assigning a new value to the variable you want to override.
+They centralize our core styles like colors, typography, spacing, sizing, and much more.
+
+Spark Design Tokens currently generate style files for web, iOS, and Android platforms.
+
+## Benefits
+Design Tokens allow more projects to incorporate our design decisions.
+
+<h3 class="docs-b-h5 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure">
+  Designer Perks
+</h3>
+
+- Encourages visual consistency and design consensus across products.
+- Creates easier syncing with brand guidelines.
+- Encourages better support for multi-platform projects.
+- Promotes clear communication with developers and other designers.
+
+<h3 class="docs-b-h5 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure">
+  Developer Perks
+</h3>
+
+- Simplifies package dependencies. React and Angular projects only need one Spark package.
+- Enables Android and iOS projects to generate style files for their products.
```

Comment:
I don't know that it would be called a style file in iOS or Android. Its more so that they have access to the vars to use in their views.